src/app/_api/data/nominations.ts

```diff
@@ -272,6 +272,24 @@ export const createNewNomination = async (
     });
   }
 
+  // call the buildbot API
+  const buildbotResponse = await fetch(`${process.env.BUILDBOT_API_URL}/webhooks/mentions`, {
```

Comment:
let's just move this into it's own file something like we have for farcaster/talent protocol. `src/app/_api/external/build_bot.ts` and import and call it here, it would be much better. I would also add a try/catch around this call in case for some reason build bot is down it does not crash creating nominations 🙏